lib/src/controller.dart

```diff
@@ -110,7 +110,7 @@ class WizardController extends ChangeNotifier {
 
   /// Requests the wizard to replace the current page with the next one.
   /// Optionally, `arguments` can be passed to the next page.
-  void replace({Object? arguments}) async {
```

Comment:
wait a sec, did we mix up something with the result and argument types? the type of arguments (input) passed to the next route, and the result (output) returned from there shouldn't be the same type.

Reply:
Should be sorted out now :)